config/index.js

```diff
@@ -0,0 +1,75 @@
+const admin = require('firebase-admin');
+admin.initializeApp();
+
+// [START validate_template]
+function validateTemplate(template) {
+  admin.remoteConfig().validateTemplate(template)
+      .then(function (validatedTemplate) {
+        // The template is valid and safe to use.
+        console.log('Template was valid and safe to use');
+      })
+      .catch(function (err) {
+        console.error('Template is invalid and cannot be published');
+        console.error(err);
+      });
+}
+// [END validate_template]
+
+// [START add_new_condition]
+function addNewCondition(template) {
+  template.conditions.push({
+    name: 'android_en',
+    expression: 'device.os == \'android\' && device.country in [\'us\', \'uk\']',
```

Comment:
If you use `"` for the string you won't have to escape the `'`:

```
"device.ios == 'android' && device.country in ['us', 'uk']"
```

Reply:
ESLint is yelling at me, should I ignore it?

Reply:
Maybe using ` instead of " will make it stop yelling?

Reply:
It is still angry 22:17 error Strings must use singlequote quotes

Reply:
Eh ok not worth fighting more, but thats strange